_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
@msecret styleguide class added in 56f4a2e.

Merge conflict resolved in 43418ae.